zhusuan/transform.py

```diff
@@ -10,10 +10,94 @@
 
 
 __all__ = [
+    'repeated_flow',
     'planar_normalizing_flow',
 ]
 
 
+def repeated_flow(base_flow_function, samples, n_iters, scope=None,
```

Comment:
Nice refactor. Thanks for the improvements.

Reply:
After we sort out correctly all of the flow distribution details, I was intending to add at least a couple of more of the flows in the literature as well.